Modules/IO/RAW/include/itkRawImageIO.hxx

```diff
@@ -101,6 +101,51 @@ void RawImageIO< TPixel, VImageDimension >
   m_ManualHeaderSize = true;
 }
 
+template < typename TStrongType >
+static void
+itkWriteRawBytesAfterSwappingMacro( const void * buffer, std::ofstream & file, ImageIOBase::ByteOrder byteOrder,
```

Comment:
This causes a warning `unused function template 'itkWriteRawBytesAfterSwappingMacro'`:
https://open.cdash.org/viewBuildError.php?type=1&buildid=6008907